src/js/BookReader.js

```diff
@@ -940,58 +966,30 @@ BookReader.prototype.drawLeafsThumbnail = function(seekIndex) {
 };
 
 BookReader.prototype.lazyLoadThumbnails = function() {
-  // We check the complete property since load may not be fired if loading from the cache
-  this.$('.BRlazyloading').filter('[complete=true]').removeClass('BRlazyloading');
-
-  var loading = this.$('.BRlazyloading').length;
-  var toLoad = this.thumbMaxLoading - loading;
-
-  var self = this;
+  const batchImageRequestByRow = async ($images) => {
+    await utils.sleep(300);
```

Comment:
_Question_ What is the `sleep` for?

Reply:
the sleep is to help give image requests some space between drawing rows instead of the current behavior - burst request of 20-40 images at a time that's putting strain on the service.